src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1723,4 +1750,5 @@ def forward(
             class_embeds=class_embeds,
             text_model_output=text_outputs,
             vision_model_output=vision_outputs,
+            objectness_logits=objectness_logits,
```

Comment:
Would be great to add a new example in the docstring of `OwlViTForObjectDetection`.